### PR TITLE
refs sparkfabrik-innovation-team/board#857: hide sensitive headers

### PR DESCRIPTION
**Pull request by Monska85**
part of sparkfabrik-innovation-team/board#857

### CODE REVIEW

File: templates/fastcgi-hide-sensitive-headers.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fastcgi_hide_header "X-Powered-By";
+fastcgi_hide_header "X-Generator";
+fastcgi_hide_header "Permissions-Policy";
```
Comment (paolomainardi) [inline templates/fastcgi-hide-sensitive-headers.conf]: we should check this one

Comment (Monska85) [reply templates/fastcgi-hide-sensitive-headers.conf]: As described [here](https://www.w3.org/TR/permissions-policy-1/), this header is a draft to give to developers the ability to select which kind of feature the app should/could claim for. I will restore this header.

**Comment by paolomainardi** [comment]
@Monska85 can i ask you to start to compile a CHANGELOG.md file ? Just to simplify the process of adding new features and pointing the teams to look at them.

**Comment by Monska85** [comment]
> @Monska85 can i ask you to start to compile a CHANGELOG.md file ? Just to simplify the process of adding new features and pointing the teams to look at them.

Added

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 2021-10-25
+
+### Added
+
+- New `NGINX_HIDE_DRUPAL_HEADERS` environment variable to hide the drupal information from the response headers (default: the headers are visible)
+- New `NGINX_HIDE_SENSITIVE_HEADERS` environment variable to hide all the sensitive information from the response headers (default: the headers will be removed)
```
Comment (paolomainardi) [inline CHANGELOG.md:12]: @mgdesign please acknowledge here, once you have backported those variables to the custom entrypoint, we can safely merge this one.

Comment (mgdesign) [reply CHANGELOG.md:12]: @paolomainardi I've opened a MR related to this change. As you merge this, we will proceed on our site. Thanks! cc @Monska85

**Comment by paolomainardi** [comment]
@mgdesign merged

**Pull request merged**